pom.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>tech.tablesaw</groupId>
+		<artifactId>tablesaw-parent</artifactId>
+		<version>0.38.3-SNAPSHOT</version>
```

Comment:
Tablesaw dependencies use 0.38.3.SNAPSHOT. They aren't available. Use 0.38.2 instead.

Reply:
Yeahs this comes from the fork, we need to find a way to support all released versions (starting with 0.38.2) while working with the snapshot to be able to release the new version at the same time. Suggestions are welcome.